programs/address-lookup-table/src/lib.rs

```diff
@@ -2,12 +2,4 @@
 #![cfg_attr(RUSTC_WITH_SPECIALIZATION, feature(specialization))]
 #![cfg_attr(RUSTC_NEEDS_PROC_MACRO_HYGIENE, feature(proc_macro_hygiene))]
 
-use solana_program::declare_id;
-
-pub mod error;
```

Comment:
can we add deprecated re-exports for the removals in this file, for the sake of backward compatibility?